Libraries/LibGfx/Color.h

```diff
@@ -401,7 +401,7 @@ class Color {
 
     constexpr Color with_opacity(float opacity) const
     {
-        return with_alpha(alpha() * opacity);
+        return with_alpha(llround(clamp(alpha() * opacity, 0.0f, 255.0f)));
```

Comment:
Why do you need `llround` if you clamp to `0..255`?

Reply:
Wouldn't `lround` do the same? I wouldn't mind `static_cast<u8>(...)` by the way, it makes it a bit more readable imo.

I guess values outside of 0..1 would be programming errors, couldn't hurt `VERIFY`ing!

Reply:
> Wouldn't lround do the same?

It would, now I just have to remember where I copied that from :sweat_smile:.